main.go

```diff
@@ -308,3 +323,22 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+// This function get the option from command argument (tlsConfig), check the validity through k8sapiflag
+// and set the config for webhook server.
+// refer to https://pkg.go.dev/k8s.io/component-base/cli/flag
+func tlsConfigSetting(cfg *tls.Config, tlsOpt tlsConfig) {
```

Comment:
This probably warrants a unit test.

Reply:
@joesbigidea i dint see any unit-test current for the main.go from makefile refer to https://github.com/aws/aws-app-mesh-controller-for-k8s/blob/master/Makefile#L35. Could u help me on this?